examples/feature-accounts/src/api/index.ts

```diff
@@ -0,0 +1,10 @@
+import { db } from "ponder:api";
+import schema from "ponder:schema";
+import { Hono } from "hono";
+import { graphql } from "ponder";
+
+const app = new Hono();
+
+app.use("/graphql", graphql({ db, schema }));
```

Comment:
Hmm - I suspect we'll want to register this at the root too for all these examples?